setup.py

```diff
@@ -822,6 +868,58 @@ def prefix(directory, files):
     'cflags': ['-DBIT_STREAM_WORD_TYPE=uint8'],
     })
 
+# SZ library and its hdf5 filter
+sz_dir = os.path.join("src", "SZ", "sz")
+sz_sources = glob(os.path.join(sz_dir, "src", "*.c"))
+sz_include_dirs = [os.path.join(sz_dir, "include"), sz_dir]
+sz_sources += glob('src/SZ/zstd/*/*.c')
+# TODO sz_depends += glob('src/SZ/zstd*/*/*.h')
+
+ZLIB_FROM_BLOSC = True
+if ZLIB_FROM_BLOSC:
+    sz_sources += glob(blosc_dir + 'internal-complibs/zlib*/*.c')
+    ##depends += glob(blosc_dir + 'internal-complibs/zlib*/*.h')
+else:
+    sz_sources += glob('src/SZ/zlib/*.c')
+
+sz_include_dirs += glob('src/SZ/zstd')
+sz_include_dirs += glob('src/SZ/zstd/common')
+
+if ZLIB_FROM_BLOSC:
```

Comment:
To be consistent with other env var to set the build, let's add a `HDF5PLUGIN_` prefix:
```suggestion
HDF5PLUGIN_ZLIB_FROM_BLOSC = True
if HDF5PLUGIN_ZLIB_FROM_BLOSC:
    sz_sources += glob(blosc_dir + 'internal-complibs/zlib*/*.c')
    ##depends += glob(blosc_dir + 'internal-complibs/zlib*/*.h')
else:
    sz_sources += glob('src/SZ/zlib/*.c')

sz_include_dirs += glob('src/SZ/zstd')
sz_include_dirs += glob('src/SZ/zstd/common')

if HDF5PLUGIN_ZLIB_FROM_BLOSC:
```